### ISSUE TITLE
Keras tuner - Incorrect parameters for model training

### ISSUE THREAD

**Issue by yiannisEfth** [tensorflow/tensorflow#47725]
URL with the issue: 
https://www.tensorflow.org/tutorials/keras/keras_tuner#instantiate_the_tuner_and_perform_hypertuning

Description of issue (what needs changing):

On the second to last code cell where the hypermodel  is re-instantiated and trained with the optimal number of epochs, the test set is incorrectly used for the fit process. The train set should be used for this process, since we then proceed to evaluate the model on the test set in the final code block.

Is the link to the source code correct?

Yes

Are all parameters defined and formatted correctly?

Yes

Are return values defined?

Yes

Are you planning to also submit a pull request to fix the issue?

Yes, here: https://github.com/tensorflow/docs/pull/1844

**Comment by yiannisEfth** [comment]
@jvishnuvardhan Already made a PR for this(?)

**Comment by jvishnuvardhan** [comment]
@yiannisEfth Sorry, I didn't notice that you have already raised a PR. 

I looked at the tutorial and noticed the typo you mentioned and updated the tutorial. My PR already merged and `master` version is already updated with the change. You can check the `master` version [here](https://github.com/tensorflow/docs/blob/master/site/en/tutorials/keras/keras_tuner.ipynb). Thanks!

I am closing this issue as the related PR got updated. As you have more updates in your PR, Please raise another issue and update the PR accordingly. Sorry for missing your PR. Thanks!

**Comment by yiannisEfth** [comment]
@jvishnuvardhan No probs! Raised a new issue and linked the PR here: https://github.com/tensorflow/tensorflow/issues/47848

**Comment by jvishnuvardhan** [comment]
@yiannisEfth Thanks for your understanding and your contribution.
